cmd/snapshot-controller/main.go

```diff
@@ -48,6 +50,9 @@ var (
 
 	leaderElection          = flag.Bool("leader-election", false, "Enables leader election.")
 	leaderElectionNamespace = flag.String("leader-election-namespace", "", "The namespace where the leader election resource exists. Defaults to the pod namespace if not set.")
+
+	metricsAddress = flag.String("metrics-address", "", "The TCP network address where the prometheus metrics endpoint will listen (example: `:8080`). The default is empty string, which means metrics endpoint is disabled.")
```

Comment:
We have a fix for the leader election library coming soon, and it's going to require a http endpoint for /healthz. Can we rename this field to be more generic like "http-endpoint"?

cc @verult who's working on the leader election fix